code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2092,3 +2092,21 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_icon_state = "planet_cracker"
 	glass_name = "Planet Cracker"
 	glass_desc = "Although historians believe the drink was originally created to commemorate the end of an important conflict in man's past, its origins have largely been forgotten and it is today seen more as a general symbol of human supremacy."
+
+/datum/reagent/consumable/ethanol/mauna_loa
+	name = "Mauna Loa"
+	description = "Extremely hot; not for the faint of heart!"
+	boozepwr = 40
+	color = "#fe8308" // 254, 131, 8
+	quality = DRINK_FANTASTIC
+	taste_description = "firey, with an aftertaste of burnt flesh"
+	glass_icon_state = "mauna_loa"
+	glass_name = "Mauna Loa"
+	glass_desc = "Lavaland, in a drink...mug...volcano...thing."
+
+/datum/reagent/consumable/ethanol/mauna_loa/on_mob_life(mob/living/carbon/M)
+	// Heats the user up while the reagent is in the body. Occasionally makes you burst into flames.
+	M.adjust_bodytemperature(10 * TEMPERATURE_DAMAGE_COEFFICIENT)
+	if (prob(1))
+		M.IgniteMob()
```

Comment:
You might wanna slip in an M.adjust_fire_stacks(1) before the M.IgniteMob() here, because IgniteMob() won't set people who don't have any fire stacks on fire.

Maybe make this a prob(5) or even a prob(10) chance (instead of a prob(1) as well?

Reply:
Alright, thanks! I wasn't really sure what firestacks did, so I hoped just using IgniteMob() would just do the trick. Also increased the prob to 5. I don't want the fire to really be that common, but looking at some other procs, it seems like the baseline for unlikely effect to be around there.